config/locales/en.yml

```diff
@@ -1915,13 +1914,32 @@ en:
       start: "Start"
       end: "End"
       activity: "Activity"
-      room: "Room"
+      no_activities: "No activities for the selected rooms/events."
+      all_venues: "All Venues"
+      room_or_stage: "Room or Stage"
+      more_details: "Show Round Details"
+      time_zone: "Time Zone"
+      rooms_panel:
+        title: "Rooms"
+        all: "All"
+        none: "None"
+        show_buttons: "Show time zone buttons"
+        use_time_zone: "Use this time zone"
       #context: when displaying the room on the general schedule, we may also display the venue if there are multiple venues
       room_in_venue: "%{room} in %{venue}"
-      venue_information_html: "You are viewing the schedule for the venue %{venue_name}."
-      timezone_message: "The schedule is displayed in the timezone %{timezone}."
-      multiple_venues_available: "This competition has multiple venues, make sure to check out the schedule for the other venues as well!"
+      venue_information_html:
+        one: "You are viewing the schedule for the venue %{venue_name}, the sole venue for this competition."
+        other: "You are viewing the schedule for the venue %{venue_name}, one of %{count} venues for this competition."
```

Comment:
Internationalize handles pluralized objects in a special way, but the `venue_information_html` key was regular and is now pluralized, so existing translations for this key cause issues during synchronization. Could you please rename this key to anything else in the English version, so that sync works properly? :)

Reply:
Sorry, this wasn't intentional! See https://github.com/thewca/worldcubeassociation.org/pull/9402

Reply:
Thanks!